### PR TITLE
Create User-Manual-for-fAIr.md

### PR DESCRIPTION
**Pull request by neelimagoogly**
I have created the User Manual for fAIr which includes step by step tutorial on how to get started with fAIr wrt issue #132 .
@petya-kangalova , @kshitijrajsharma and @omranlm please review it.

### CODE REVIEW

**Comment by kshitijrajsharma** [comment]
HI @neelimagoogly , Thanks for the PR ! 
Its a good practice to use Github Keywords in issues/ PR  https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests . For example for This PR 
It could be 
- Resolve #132 

Here ,  this resolve keyword with issue which is  resolved / contributed by PR will be closed automatically when PR is reviewed and merged !  
And also one suggestion on branching and commit messages ( I also personally need to work on commit msgs though haha ) 
We use ```[commitizen](https://commitizen-tools.github.io/commitizen/)```  , It would be easy and good on long run if you have proper naming for your branch and commit . In long term  People may forget and try to checkout the work done or even for the developer itself it would be hard to track and manage branches if it is patch 1 patch 2 if you name it differently then will be easy to follow  , for eg : for this PR it could be ```docs/tutorial```

**Comment by neelimagoogly** [comment]
> 

Thanks a lot @kshitijrajsharma . I would definitely follow this practice onwards .

File: docs/User-Manual-for-fAIr.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
+# User Manual for FMTM
```
Comment (kshitijrajsharma) [inline docs/User-Manual-for-fAIr.md:1]: User Manual for fAIr

**Review by kshitijrajsharma — APPROVED** [review]

**Comment by petya-kangalova** [comment]
@kshitijrajsharma @omranlm  I know this PR has been merged, can we now link the content to this page https://fair-dev.hotosm.org/documentation ? I can then also build an FAQ and link to documentation and we can even add some of the videos/tutorials there.

**Pull request merged**